xblock/reference/plugins.py

```diff
@@ -171,3 +130,42 @@ def __set__(self, xblock, value):
         We don't support this until we figure out what this means
```

Comment:
StackOverflow has several good articles explaining the `__set__` method and there are several nice howtos on Python descriptors.